tests/system/small/test_pandas_options.py

```diff
@@ -288,9 +287,7 @@ def test_close_session_after_credentials_need_reauthentication(monkeypatch):
         with pytest.raises(google.auth.exceptions.RefreshError):
             bpd.read_gbq(test_query)
 
-        # Now verify that closing the session works
-        bpd.close_session()
-        assert bigframes.core.global_session._global_session is None
```

Comment:
Please find a way to restore this. If folks aren't authenticated properly I'd like to provide a way to at least restore the client-side state.

Let's raise a warning with the session ID saying that we weren't able to clean up.

Reply:
I have changed close_session to just warn instead of error out when it hits the credentials expired error. The test now checks to make sure there's a warning.